conn.go

```diff
@@ -463,7 +473,7 @@ func (hc *halfConn) encrypt(b *block, explicitIVLen int) (bool, alert) {
 			payloadLen := len(b.data) - recordHeaderLen - explicitIVLen
 			overhead := c.Overhead()
 			if hc.version >= VersionTLS13 {
-				overhead++
+				overhead++ // TODO(kk): why this is done?
```

Comment:
I believe it's because of this: "The length (in bytes) of the following TLSCiphertext.encrypted_record, which is the sum of the lengths of the content and the padding, **plus one for the inner content type**, plus any expansion added by the AEAD algorithm."

Reply:
That's right.

Reply:
so comment is needed